src/V3AstNodeExpr.h

```diff
@@ -482,6 +500,15 @@ class AstNodeUniop VL_NOT_FINAL : public AstNodeExpr {
     bool isPure() override;
     const char* broken() const override;
 };
+class AstNodeFUniop VL_NOT_FINAL : public AstNodeUniop {
+    // File uniop expression
+protected:
+    AstNodeFUniop(VNType t, FileLine* fl, AstNodeExpr* lhsp)
+        : AstNodeUniop(t, fl, lhsp) {}
+
+public:
+    ASTGEN_MEMBERS_AstNodeFUniop;
+};
```

Comment:
It doesn't seem worth adding this base for just 2 usages.  Actually I think you're better off having a virtual function indicating if coverage is allowed on this node type, and setting appropriately in whatever shouldn't use it.  It's possible you can just default it to the same as isGateOptimizaable and that will get most or all of the cases.